morpheus/cli/stage_registry.py

```diff
@@ -30,13 +31,15 @@ class StageInfo:
     modes: typing.List[PipelineModes]
     qualified_name: str  # The fully qualified name of the stage. Only used for comparison
     build_command: typing.Callable[[], click.Command] = dataclasses.field(compare=False, repr=False)
+    get_stage_class: typing.Callable[[], typing.Type[_pipeline.StreamWrapper]] = dataclasses.field(compare=False,
```

Comment:
Understand the need but this adds extra overhead to testing that could be accomplished by using `imporlib` like the plugin manager does